Tools/autotest/autotest.py

```diff
@@ -328,40 +328,31 @@ def write_fullresults():
     results.addglob('DataFlash Log', '*-log.bin')
     results.addglob("MAVLink log", '*.tlog')
     results.addglob("GPX track", '*.gpx')
-    results.addfile('ArduPlane build log', 'ArduPlane.txt')
-    results.addfile('ArduPlane code size', 'ArduPlane.sizes.txt')
-    results.addfile('ArduPlane stack sizes', 'ArduPlane.framesizes.txt')
-    results.addfile('ArduPlane defaults', 'default_params/ArduPlane-defaults.parm')
-    results.addglob("ArduPlane log", 'ArduPlane-*.BIN')
-    results.addglob("ArduPlane core", 'ArduPlane.core')
-    results.addglob("ArduPlane ELF", 'ArduPlane.elf')
-    results.addfile('ArduCopter build log', 'ArduCopter.txt')
-    results.addfile('ArduCopter code size', 'ArduCopter.sizes.txt')
-    results.addfile('ArduCopter stack sizes', 'ArduCopter.framesizes.txt')
-    results.addfile('ArduCopter defaults', 'default_params/ArduCopter-defaults.parm')
-    results.addglob("ArduCopter log", 'ArduCopter-*.BIN')
-    results.addglob("ArduCopter core", 'ArduCopter.core')
-    results.addglob("ArduCopter elf", 'ArduCopter.elf')
+
+    # results common to all vehicles:
+    vehicle_files = [ ('%(vehicle)s build log', '%(vehicle)s.txt'),
+                      ('%(vehicle)s code size', '%(vehicle)s.sizes.txt'),
+                      ('%(vehicle)s stack sizes', '%(vehicle)s.framesizes.txt'),
+                      ('%(vehicle)s defaults', 'default_params/%(vehicle)s-defaults.parm'),
+                      ('%(vehicle)s core', '%(vehicle)s.core'),
+                      ('%(vehicle)s ELF', '%(vehicle)s.elf'),
```

Comment:
Nitpicking: using `{vehicle}` which is the new Python style looks better (and using `.format` later too).

Reply:
Hmph.  I changed it away from that :-)

Moved it back...